bindings/nodejs/src/lib.rs

```diff
@@ -292,6 +310,102 @@ impl BlockingLister {
     }
 }
 
+#[napi]
+pub struct Reader(opendal::Reader);
+
+#[napi]
+impl Reader {
+    /// # Safety
+    ///
+    /// > &mut self in async napi methods should be marked as unsafe
+    ///
+    /// napi will make sure the function is safe, and we didn't do unsafe
+    /// thing internally.
+    #[napi]
+    pub async unsafe fn read(&mut self, length: Option<i64>) -> Result<Buffer> {
```

Comment:
Are we unable to implement `ReadableStream` at the moment? If that's the case, I would rather not include support for the `reader` API in Node.js bindings.

Reply:
It seems like the Stream implementation hasn't been done yet. So, it has to be put on hold for now.